### PR TITLE
Implement Disposable Workspace in all Importers

### PR DESCRIPTION
**Pull request by tkukuljan**
#64

### CODE REVIEW

**Comment by tkukuljan** [comment]
The pull request is finished by Andrea and me.

FYI: @dcolleoni @dtrzesniak @amuolo

File: ifrs17/Import/Importers.ipynb [hunk 1]
```diff
@@ -358,11 +431,11 @@
         {
             "cell_type": "markdown",
             "source": [
-                "The [Analysis of Change configuration](../DataModel/DataStructure#aoc-step-configuration) is parsed from the input file and complemented with defaults to allow for an easy insertion of new AoC steps. ",
+                "The [Analysis of Change configuration](../DataModel/DataStructure#aoc-step-configuration) is parsed from the input file and complemented with defaults to allow for an easy insertion of new Aoc steps. ",
```
Comment (dcolleoni) [inline ifrs17/Import/Importers.ipynb:434]: AoC Steps

File: ifrs17/Import/Importers.ipynb [hunk 2]
```diff
@@ -358,11 +431,11 @@
...
                 "\n",
                 "\nAfter having checked that the [AocTypes](../DataModel/DataStructure#aoc-variable-type) loaded in the target DataSource are including all the compulsory ones, default configurations are generated on the basis of the AocTypes ordering. ",
                 "\n",
-                "\nThe following categories have been identified based on the *Order* of the novel AoC Step:",
+                "\nThe following categories have been identified based on the *Order* of the novel Aoc step:",
```
Comment (dcolleoni) [inline ifrs17/Import/Importers.ipynb:438]: revert

File: ifrs17/Import/Importers.ipynb [hunk 3]
```diff
@@ -373,7 +446,7 @@
                 "\n| CRU < Order < WO      |  EV with Novelty I and N                              |",
                 "\n| WO  < Order < CL      |  WO with Novelty C (only for Import Source = Actual)  |",
                 "\n",
-                "\nThe new AoC Configurations are created with the same order of the AoC Types. "
+                "\nThe new Aoc Configurations are created with the same order of the Aoc Types. "
```
Comment (dcolleoni) [inline ifrs17/Import/Importers.ipynb:449]: revert

**Review by dcolleoni — CHANGES_REQUESTED** [review]
You are reverting many changes that were done in previous PR. 
You have most probably resolved conflicts in the wrong way. 
Thanks for fixing this.

**Review by dcolleoni — APPROVED** [review]

**Pull request merged**